tests/Carbon/AddTest.php

```diff
@@ -183,7 +183,7 @@ public function testAddDayPassingArg()
 
     public function testAddHourPassingArg()
     {
-        $this->assertSame(2, Carbon::createFromTime(0)->addHour(2)->hour);
+        $this->assertSame(2, Carbon::createFromTime(0, 0, 0, 'UTC')->addHour(2)->hour);
```

Comment:
This will do the same, set minutes and seconds to zero:

https://github.com/briannesbitt/Carbon/blob/bba6c6e410c6b4317e37a9474aeaa753808c3875/src/Carbon/Carbon.php#L577

Reply:
No, it's all about the TimeZone. When running this test on Windows, it's failed like:
```php
date_default_timezone_set('Asia/Ho_Chi_Minh');
$d = new \DateTime('2018-03-11 00:00:00', new \DateTimeZone('America/Toronto'));
echo $d->modify('+2 hour')->format('G'); // output: 3
```

Reply:
DST problem, I understand. Nice catch.